File: WoofAndroid/presentation/account/src/main/kotlin/com/perrystreet/woof/presentation/account/uimodel/AccountUIModel.kt
package com.perrystreet.woof.presentation.account.uimodel

data class AccountUIModel(
    val name: String,
    val photoUrl: String,
    val summary: AccountSummaryUIModel,
    val bio: String,
    val personality: List<String>,
    val rows: List<AccountDetailRowUIModel>,
)

File: WoofIOS/SwiftPackages/Presentation/Account/Sources/PresentationAccount/UIModel/AccountUIModel.swift
import Foundation

public struct AccountUIModel: Hashable {
    public let name: String
    public let photoUrl: String
    public let summary: AccountSummaryUIModel
    public let bio: String
    public let personality: [String]
    public let rows: [AccountDetailRowUIModel]

    public init(
        name: String,
        photoUrl: String,
        summary: AccountSummaryUIModel,
        bio: String,
        personality: [String],
        rows: [AccountDetailRowUIModel]
    ) {
        self.name = name
        self.photoUrl = photoUrl
        self.summary = summary
        self.bio = bio
        self.personality = personality
        self.rows = rows
    }
}
